src/selectors/entities/channels.js

```diff
@@ -1156,3 +1156,44 @@ export const getSortedDirectChannelWithUnreadsIds: (GlobalState, Channel, boolea
         return filterChannels(unreadChannelIds, favoritePreferences, directChannelIds, false, favoritesAtTop);
     },
 );
+
+export const getDefaultChannelForTeams: (GlobalState) => RelationOneToOne<Team, Channel> = createSelector(
```

Comment:
Yes, exactly, I'm not replacing the concept entirely, because the default channel will exist anyway, I'm adding the RedirectChannel in a lot of places as the concept to use, instead the DEFAULT_CHANNEL, but the default channel is still there, in most of the cases under the redirect channel, and in some cases is used as is (for example for read only default channel, and things like that).